src/devices/philips.ts

```diff
@@ -135,6 +135,13 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Hue white ambiance",
         extend: [philips.m.light({colorTemp: {range: [222, 454]}})],
     },
+    {
+        zigbeeModel: ["LTA019"],
+        model: "LTA019",
```

Comment:
Please use the model number written on the Philips site here (also rename the `.png` accordingly)